Doc/library/typing.rst

```diff
@@ -1780,9 +1780,8 @@ These are not used in annotations. They are building blocks for declaring types.
 
       Point2D = TypedDict('Point2D', x=int, y=int, label=str)
 
-   .. deprecated-removed:: 3.11 3.13
-      The keyword-argument syntax is deprecated in 3.11 and will be removed
-      in 3.13. It may also be unsupported by static type checkers.
+   .. versionchanged:: 3.13
+      Removed support for the keyword-argument method of creating ``TypedDict``.
```

Comment:
Could you move this note down below the other `.. versionchanged` notes? I.e., just below this line here:

https://github.com/python/cpython/blob/fe77a99fc8b549a8bf9ccbc5485fe5ea9bcf47b9/Doc/library/typing.rst?plain=1#L1943-L1944